download.jl

```diff
@@ -0,0 +1,82 @@
+#Extracting wind data from MET file 
+download("https://earthsciml.s3.amazonaws.com/compressed_Input/met/mcipv5/METDOT3D_160701.bson")
+
+#Change the name after downloading the file from AWS. 
+#The directories will vary by users 
+
+file = "/var/folders/s3/d0ks9vxn08dclbqqy0q3tgzw0000gn/T/jl_MRWMZx"
+mv(file, "METDOT3D_160701.bson")
```

Comment:
I don't think we need to delete this file, can just leave it in the original locaiotn.